R/assertions.R

```diff
@@ -1512,29 +1514,23 @@ assert_expr_list <- function(arg, # nolint
 #' fulfilling the condition are listed.
 #'
 #' @param list A list to be checked
-#'
 #'   A list of named lists or classes is expected.
-#'
 #' @param element The name of an element of the lists/classes
-#'
 #'   A character scalar is expected.
-#'
 #' @param condition Condition to be fulfilled
-#'
 #'   The condition is evaluated for each element of the list. The element of the
 #'   lists/classes can be referred to by its name, e.g., `censor == 0` to check
 #'   the `censor` field of a class.
-#'
-#' @param message_text Text to be displayed in the message
-#'
-#'   The text should describe the condition to be fulfilled, e.g., "For events
-#'   the censor values must be zero.".
-#'
+#' @param message_text Text to be displayed in the error message above
+#'   the examples of elements that do not meet the condition.
```

Comment:
sorry - what are the examples of elements?

Reply:
Ha, this is what I wrote after adding more details, because I didn't understand the original text! I updated to "Text to be displayed in the error message above the listing of values that do not meet the condition."